charts/kafka-ui/templates/ingress.yaml

```diff
@@ -81,7 +81,7 @@ spec:
               servicePort: {{ .servicePort }}
           {{- end }}
 {{- if .Values.ingress.host }}
-      host: {{ .Values.ingress.host }}
+      host: {{ tpl .Values.ingress.host . }}
```

Comment:
Hi, thank you for the contribution.

Could you please add the same structure to the same param reference on the line 62 as well to  standardize things and to avoid problems for kubernetes versions above 1.22?